editor/block-settings-menu/style.scss

```diff
@@ -35,7 +49,7 @@
 		display: block;
 	}
 
-	&:first-child {
-		margin-bottom: 4px;
+	.editor-block-settings-menu__content &:first-child {
```

Comment:
It looks like this can be moved to line 17. We have rules for `.editor-block-settings-menu__content` there.

Reply:
Don't you think it makes more sense to keep it in `.editor-block-settings-menu__control` because it's applied to this class when it's first on the list? I can see arguments for both which makes the choice difficult.

Reply:
It takes some time to process those class names either way. CSS is complicated. We could add another class name, but having a section `.editor-block-settings-menu__elipsis` just to change the margin is most likely be too much here.